src/main/java/de/tum/in/www1/artemis/web/rest/ExamResource.java

```diff
@@ -1321,4 +1323,19 @@ public ResponseEntity<Set<SuspiciousExamSessionsDTO>> getAllSuspiciousExamSessio
                 analyzeSessionsIpOutsideOfRange);
         return ResponseEntity.ok(examSessionService.retrieveAllSuspiciousExamSessionsByExamId(examId, options, Optional.ofNullable(ipSubnet)));
     }
+
+    /**
+     * GET /courses/{courseId}/exams/{examId}/deletion-summary : Get a summary of the deletion of an exam.
+     *
+     * @param courseId the id of the course
+     * @param examId   the id of the exam
```

Comment:
I think we can keep it shorter here or add more information that is not already contained in the variable name
```suggestion
     * @param courseId
     * @param examId
```

Reply:
Same, client style.